saskatoon/member/admin.py

```diff
@@ -386,6 +426,21 @@ def contact(self, org):
             return mark_safe(f"<a href={url}>{obj}</a>")
         return None
 
+    def save_related(self, request, form, formsets, change):
+        """Ensure is_equipment_point box was not mistakenly left checked/unchecked"""
+
+        super().save_related(request, form, formsets, change)
+
+        org = Organization.objects.get(actor_id=form.instance.actor_id)
+        if not org.equipment.exists():
+            org.is_equipment_point = False
+            org.save()
```

Comment:
edge case, but would it make sense to allow setting `is_equipment_point` without having equipment?

e.g. if an org is found to be an equipment point, but nobody has yet to find out what equipment they actually have. the toggle would be useful for pick leaders to look into it, especially if the org a lot closer to the harvest than others.

Reply:
The current mechanism to "book" and equipment point is to store all of its equipment in the `equipment_reserved` field so if an organization has no equipment it cannot be booked in practice. I would much rather we create a dummy equipment type if needed

Reply:
I will add a warning message when `is_equipmen_point` is overriden on behalf of the user